sdk/examples/client/block/02_block_custom_parents.rs

```diff
@@ -10,7 +10,8 @@ use iota_sdk::client::{Client, Result};
 async fn main() -> Result<()> {
     // Take the node URL from command line argument or use one from env as default.
     let node_url = std::env::args().nth(1).unwrap_or_else(|| {
-        // This example uses dotenv, which is not safe for use in production.
+        // This example uses secrets stored in environment variables for simplicity sake but should generally not be
+        // done in production.
```

Comment:
Can you make it shorter so it fits one line pls?

Reply:
I'd like as less distraction as possible in the example, let's make this straight to the point

Reply:
Also please remove stored, I don't think it really applies well here